packages/wrangler/src/pages/dev.ts

```diff
@@ -144,6 +148,11 @@ export function Options(yargs: CommonYargsArgv) {
 				type: "array",
 				description: "R2 bucket to bind (--r2 R2_BINDING)",
 			},
+			service: {
+				type: "array",
+				description: "Service to bind (--service SERVICE=WORKER_NAME[ENVIRONMENT])",
```

Comment:
```suggestion
				description: "Service to bind (--service SERVICE=SCRIPT_NAME)",
```

I'd like to avoid calling out the environment bit until we figure out something a bit more holistic here. For example, how can I bind to a DO in another script which is using a different environment?

Reply:
I see... ok let's remove it

also `SCRIPT_NAME` feels very awkward to me here... but yeah I guess for consistency with the other args we can use that 😓